### PR TITLE
Ldm unet convert fix

### PR DESCRIPTION
**Pull request by DN6**
# What does this PR do?

Fixes # (issue)
#5917

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py [hunk 1]
```diff
@@ -381,7 +386,12 @@ def create_ldm_bert_config(original_config):
 
 
 def convert_ldm_unet_checkpoint(
-    checkpoint, config, path=None, extract_ema=False, controlnet=False, skip_extract_state_dict=False
+    checkpoint,
+    config,
+    path=None,
+    extract_ema=False,
+    controlnet=False,
+    skip_extract_state_dict=False,
```
Comment (sayakpaul) [inline src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py:394]: Same as above.

File: src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py [hunk 2]
```diff
@@ -497,17 +507,31 @@ def convert_ldm_unet_checkpoint(
             )
 
         paths = renew_resnet_paths(resnets)
-        meta_path = {"old": f"input_blocks.{i}.0", "new": f"down_blocks.{block_id}.resnets.{layer_in_block_id}"}
+        meta_path = {
+            "old": f"input_blocks.{i}.0",
+            "new": f"down_blocks.{block_id}.resnets.{layer_in_block_id}",
+        }
```
Comment (sayakpaul) [inline src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py:513]: Same.

File: src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py [hunk 3]
```diff
@@ -497,17 +507,31 @@ def convert_ldm_unet_checkpoint(
...
         assign_to_checkpoint(
-            paths, new_checkpoint, unet_state_dict, additional_replacements=[meta_path], config=config
+            paths,
+            new_checkpoint,
+            unet_state_dict,
+            additional_replacements=[meta_path],
+            config=config,
```
Comment (sayakpaul) [inline src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py:519]: Same.

File: src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py [hunk 4]
```diff
@@ -497,17 +507,31 @@ def convert_ldm_unet_checkpoint(
...
         )
 
         if len(attentions):
             paths = renew_attention_paths(attentions)
 
-            meta_path = {"old": f"input_blocks.{i}.1", "new": f"down_blocks.{block_id}.attentions.{layer_in_block_id}"}
+            meta_path = {
+                "old": f"input_blocks.{i}.1",
+                "new": f"down_blocks.{block_id}.attentions.{layer_in_block_id}",
```
Comment (sayakpaul) [inline src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py:527]: Same.

File: src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py [hunk 5]
```diff
@@ -497,17 +507,31 @@ def convert_ldm_unet_checkpoint(
...
+            }
             assign_to_checkpoint(
-                paths, new_checkpoint, unet_state_dict, additional_replacements=[meta_path], config=config
+                paths,
+                new_checkpoint,
+                unet_state_dict,
+                additional_replacements=[meta_path],
+                config=config,
```
Comment (sayakpaul) [inline src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py:534]: Same.

**Review by sayakpaul — APPROVED** [review]
LGTM. I think the formatting related changes could be reverted.

**Comment by patrickvonplaten** [comment]
@DN6 there seems to be something off with your linter as it's always changing every file that is adapted no?

**Comment by DN6** [comment]
@patrickvonplaten I fixed it. Think this branch was created before I applied the fix. Will resolve now

**Pull request merged**